plugins/flp_import/CMakeLists.txt

```diff
@@ -3,4 +3,8 @@ INCLUDE(BuildPlugin)
 INCLUDE_DIRECTORIES(unrtf)
 
 ADD_DEFINITIONS(--std=c++0x)
+IF(HOMEBREW)
+	ADD_DEFINITIONS(-stdlib=libc++)
```

Comment:
Done via https://github.com/tresf/lmms/commit/ad4b764ec6f677f36371081406a511e82c9c41da

Note, I removed the double quotes for consistency and removed the redundant dash in flp_import as well.  Hope that is ok. :)